<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit_user_form</class>
 <widget class="QWidget" name="Edit_user_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit_user_form</string>
  </property>
  <widget class="QFrame" name="frame_window">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>356</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame[objectName=&quot;frame_window&quot;]
{
	border-radius:15px;
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>0</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	border-top-right-radius:14px;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(255, 0, 0);
}

</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>356</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;
color:rgb(20,20,20);
padding-bottom:1px;
border-top-left-radius:14px;
border-top-right-radius:14px;
background-color: rgb(110, 205, 255)</string>
    </property>
    <property name="text">
     <string>修改资料</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>85</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_birthday">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>175</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>生日：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>85</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sex">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_male">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>130</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13px</string>
    </property>
    <property name="text">
     <string>男</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_female">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13px</string>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="de_birthday">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>175</y>
      <width>126</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
border:1px solid white;
border-left-color: rgba(255, 255, 255,0);
border-right-color: rgba(255, 255, 255,0);
border-top-color: rgba(255, 255, 255,0);
border-bottom-color: rgb(85, 170, 255);
font: 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>215</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>215</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_re_password">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>265</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>重复密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_re_password">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>265</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>45</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>45</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>lbl_title</zorder>
   <zorder>btn_close</zorder>
   <zorder>lbl_name</zorder>
   <zorder>lbl_birthday</zorder>
   <zorder>le_name</zorder>
   <zorder>lbl_sex</zorder>
   <zorder>rb_male</zorder>
   <zorder>rb_female</zorder>
   <zorder>de_birthday</zorder>
   <zorder>lbl_password</zorder>
   <zorder>le_password</zorder>
   <zorder>lbl_re_password</zorder>
   <zorder>le_re_password</zorder>
   <zorder>lbl_username</zorder>
   <zorder>le_username</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Edit_user_form</receiver>
   <slot>btn_close_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_close_click()</slot>
 </slots>
</ui>
